packages/edit-post/src/components/sidebar/settings-header/index.js

```diff
@@ -27,66 +33,68 @@ const SettingsHeader = ( { sidebarName } ) => {
 		};
 	}, [] );
 
-	const [ documentAriaLabel, documentActiveClass ] =
+	const [ documentAriaLabel ] =
 		sidebarName === 'edit-post/document'
 			? // translators: ARIA label for the Document sidebar tab, selected. %s: Document label.
 			  [ sprintf( __( '%s (selected)' ), documentLabel ), 'is-active' ]
 			: [ documentLabel, '' ];
 
-	const [ blockAriaLabel, blockActiveClass ] =
+	const [ blockAriaLabel ] =
 		sidebarName === 'edit-post/block'
 			? // translators: ARIA label for the Block Settings Sidebar tab, selected.
 			  [ __( 'Block (selected)' ), 'is-active' ]
 			: // translators: ARIA label for the Block Settings Sidebar tab, not selected.
 			  [ __( 'Block' ), '' ];
```

Comment:
Since we're not destructuring the active class anymore, we can also remove it from the lines below

```suggestion
	const blockAriaLabel =
		sidebarName === 'edit-post/block'
			? // translators: ARIA label for the Block Settings Sidebar tab, selected.
			  __( 'Block (selected)' )
			: // translators: ARIA label for the Block Settings Sidebar tab, not selected.
			  __( 'Block' );
```

Same for the `templateAriaLabel` below